test/e2e/e2e_suite_test.go

```diff
@@ -40,7 +39,7 @@ func TestMain(m *testing.M) {
 	// Now that we know which Viper config (if any) was chosen,
 	// parse it and update those options which weren't already set via command line flags
 	// (which have higher priority).
-	if err := viperconfig.ViperizeFlags(*viperConfig, "e2e", flag.CommandLine); err != nil {
+	if err := viperizeFlags(*viperConfig, "e2e", flag.CommandLine); err != nil {
```

Comment:
I think that we can just drop it, we don´t use it for these tests

Reply:
Ah sweet, I was not sure about this.